ia/src/protocole.py

```
@@ -115,11 +96,20 @@
             message, buffer = buffer.split("\n", 1)
 
             if message:
-                if (message == "dead\n"):
+                if (message == "dead"):
                     print("Client dead, closing connection")
                     handle_Dead(client, message)
-                if (message and client["commandes"]):
-                    handle_commande(client, client["commandes"][0], message)
+
+                if (client["commandes"]):
```

Pylint (3.3.7)

[convention] Unnecessary parens after 'if' keyword

(C0325)